src/graph.cpp

```diff
@@ -328,6 +328,99 @@ void Graph::augmentMinCostPath(std::list<Edge *> edges, const unsigned int &valu
 }
 
 
+/**
+ * Takes a number and sets the bool "selected" of that amount of edges, their reverse edges and their corresponding edges in the residual graph to false
```

Comment:
Make it explicit that this is randomized, both in documentation and maybe method name.

Reply:
This is not my function. but fine